hw/ip/entropy_src/doc/theory_of_operation.md

```diff
@@ -16,7 +16,14 @@ When entropy is delivered to the downstream hardware block, a signal will indica
 Once the initial boot-time mode phase has completed, the ENTROPY_SRC block can be switched to FIPS/CC compliant mode (for simplicity referred to as FIPS mode) by setting the `FIPS_ENABLE` field in the [`CONF`](registers.md#conf) register to `kMultiBitBool4True`.
 In this mode, once the raw entropy has been health checked, it will be passed into a conditioner block.
 This block will compress the bits such that the entropy bits/physical bits, or min-entropy value, should be improved over the raw data source min-entropy value.
-The compression operation, by default, will compress every 2048 tested bits into 384 full-entropy bits.
+The compression operation will compress every [`HEALTH_TEST_WINDOWS.FIPS_WINDOW`](registers.md#health_test_windows--fips_window) x 4 tested bits into 384 full-entropy bits.
+By default, 2048 tested bits are used.
+Note that after enabling, 1024 samples of 4 bits need to be used for startup health testing.
+These bits are used for producing the first seed, i.e., 4096 tested bits or twice as much are used for the first seed.
+In addition, a seed is only produced if the last [`HEALTH_TEST_WINDOWS.FIPS_WINDOW`](registers.md#health_test_windows--fips_window) x 4 tested bits have not failed the health tests.
```

Comment:
Nit: It might be easier to read without the double negative here, so maybe "if ... have passed the health tests" ?

Reply:
Yes, that makes sense.